webapp/channels/src/components/widgets/menu/menu_items/menu_start_trial.tsx

```diff
@@ -84,20 +85,24 @@ const MenuStartTrial = (props: Props): JSX.Element | null => {
             id={props.id}
         >
             <FreeVersionBadge>{'FREE EDITION'}</FreeVersionBadge>
-            {isE0 &&
-                <div className='start_trial_content'>
-                    {formatMessage({
-                        id: 'navbar_dropdown.versionTextE0',
-                        defaultMessage: 'This is the free edition of Mattermost, ideal for evaluation and small teams.',
-                    })}
-                </div>}
-            {!isE0 &&
-                <div className='start_trial_content'>
-                    {formatMessage({
-                        id: 'navbar_dropdown.versionTextTeamEdition',
-                        defaultMessage: 'This is the free open source edition of Mattermost, ideal for evaluation and small teams.',
-                    })}
-                </div>}
+            <div className='editionText'>
+                {formatMessage(
+                    {
+                        id: 'navbar_dropdown.versionText',
+                        defaultMessage: 'This is the free <link>unsupported</link> edition of Mattermost.',
+                    },
+                    {
+                        link: (msg: React.ReactNode) => (
+                            <ExternalLink
+                                location='unsupported-link'
```

Comment:
I generally try to be more specific with the location so we can determine where exactly this was hit - something like `menu_start_trial.unsupported-link`

Reply:
Done